service/controller/clusterapi/v30/cloudconfig/config.go

```diff
@@ -0,0 +1,104 @@
+package cloudconfig
+
+import (
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+
+	"github.com/giantswarm/aws-operator/service/controller/clusterapi/v30/encrypter"
+)
+
+type Config struct {
```

Comment:
The config is exactly equal for all rendering implementations. We extract it to not duplicate the complexity of the defaulting and validation.